server/start.go

```diff
@@ -20,15 +20,16 @@ import (
 
 // Tendermint full-node start flags
 const (
-	flagWithTendermint  = "with-tendermint"
-	flagAddress         = "address"
-	flagTraceStore      = "trace-store"
-	flagPruning         = "pruning"
-	flagCPUProfile      = "cpu-profile"
-	FlagMinGasPrices    = "minimum-gas-prices"
-	FlagHaltHeight      = "halt-height"
-	FlagHaltTime        = "halt-time"
-	FlagInterBlockCache = "inter-block-cache"
+	flagWithTendermint    = "with-tendermint"
+	flagAddress           = "address"
+	flagTraceStore        = "trace-store"
+	flagPruning           = "pruning"
+	flagCPUProfile        = "cpu-profile"
+	FlagMinGasPrices      = "minimum-gas-prices"
+	FlagHaltHeight        = "halt-height"
+	FlagHaltTime          = "halt-time"
+	FlagInterBlockCache   = "inter-block-cache"
+	FlagUnsafeSkipUpgrade = "unsafe-skip-upgrade"
```

Comment:
```suggestion
	FlagUnsafeSkipUpgrades = "unsafe-skip-upgrades"
```

Since we can provide a set/list of heights ;-)

Reply:
Done